Makefile

```diff
@@ -30,6 +30,6 @@ install:
 	cd ansible_collections ; ansible-galaxy collection install $B -f
 
 unit_test: clean $B install
-	cd ansible_collections/$N/$M ; ansible-test units -vvvvv --python 3.9 --local
```

Comment:
Why ?
FYI:  removing it break things on my machine:
  /usr/bin/python2.7: No module named pytest

Reply:
I think we really shouldn't hard code the Python version here because then we lock our tests to be run with the exact version (3.10 won't work, etc.).

So something is fishy with your Python environment, I think, as it should discover the latest python version automatically. Feel free to ping me so we can work out the best way to fix it. (may be adding some additional processing here for prefix installs...)